### DISCUSSION TITLE
isolcpus for TX2

### DISCUSSION THREAD

**Discussion by lfdmn** [OE4T/meta-tegra#550] [General]
Hi,

On dunfell branch ```conf/machine/include/tegra186.inc``` contains ```isolcpus=1-2``` in ```KERNEL_ARGS```. 

```
KERNEL_ARGS ?= "console=ttyS0,115200 console=tty0 fbcon=map:0 isolcpus=1-2"
```
Is this intentional?

**Comment by madisongh** [comment]
Yes.  It's directly copied from the kernel command line settings in the L4T BSP for the TX2.  Those are the Denver cores, and IIRC there have been some performance issues with those cores that led them to disabling them by default.  You should find some posts in the NVIDIA dev forum about ti.
